aliBuild

```diff
@@ -237,10 +247,10 @@ class HttpRemoteSync:
       else:
         hashList = json.loads(urlopen(hashListUrl).read())
         pkgList = json.loads(urlopen(pkgListUrl).read())
-    except URLError, e:
+    except URLError as e:
```

Comment:
Correct. It's just to simplify merging. As I said I suspect you will have conflicts as I refactored a bit aliDoctor to improve user experience and to make it behave exactly like aliBuild